### ISSUE TITLE
crash in tracker3-miners results in 'Retrace failed. Try again later and if the problem persists report this issue please.'

### ISSUE THREAD

**Issue by cmurf** [abrt/retrace-server#411]
Attaching `abrt-cli report -vvv` but it really isn't revealing why it failed.

```
Uploading 5.8 MiB
Upload successful
Retrace job started
Generating backtrace
Retrace job failed
Retrace failed. Try again later and if the problem persists report this issue please.
[2021-01-31 00:21:36] [I] Analyzing crash data
```
[retracefailed.txt](https://github.com/abrt/retrace-server/files/5899004/retracefailed.txt)

**Comment by msrb** [comment]
@mgrabovsky any idea if the retrace server had some outage or something?

**Comment by mgrabovsky** [comment]
I'm not aware of an outage at that time.

**Comment by cmurf** [comment]
I did try multiple times over about 30 minutes; the local backtrace was successfully used to file a new rhbz.

**Comment by mgrabovsky** [comment]
@cmurf Yeah, this seems to be a bug with retracing certain packages. Could you please share the link to the created bug report or upload the dump directory? The information contained therein might shed some light on why this particular problem directory is— well, problematic.

**Comment by cmurf** [comment]
Looks like a new bug was not filed, there was a dup and it just cc'd me on the dup.
https://bugzilla.redhat.com/show_bug.cgi?id=1909106
